src/main/kotlin/org/jetbrains/exposed/dao/UUIDEntity.kt

```diff
@@ -0,0 +1,15 @@
+package org.jetbrains.exposed.dao
+
+import org.jetbrains.exposed.sql.Column
+import java.util.UUID
+
+open class UUIDTable(name: String = "", columName: String = "id") : IdTable<String>(name) {
+  override val id: Column<EntityID<String>> = varchar(columName, 36)
```

Comment:
I agree, it needs to be consistent. I just didn't have time to dig into the depths and created this for that purpose, I have it implemented in my project but would rather see it running native.